### ISSUE TITLE
GEOS-IT ExtData reading error and debugging

### ISSUE THREAD

**Issue by yuanjianz** [geoschem/gchp#379]
### Name and Institution (Required)

Name: Yuanjian Zhang
Institution: Washington University in St. Louis

### Confirm you have reviewed the following documentation

- [x] [Support guidelines](https://gchp.readthedocs.io/en/latest/reference/SUPPORT.html)
- [x] [User manuals](https://gchp.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [x] [Debugging GEOS-Chem and HEMCO errors](https://gchp.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [x] [Current and past Github issues](https://github.com/geoschem/GCHP/issues)

### Description of your issue or question

I am trying to use the newly updated GCHP 14.3 for GEOS-IT meteorological field implementation. While using GEOS-IT Native C180 and 0.5x0.625, both cases occurred run time error when reading ExtData.

```
Choose meteorology source:
-----------------------------------------------------------
  1. MERRA-2 (Recommended)
  2. GEOS-FP
  3. GEOS-IT (Beta release)
>>> 3
Choose meteorology files:
-----------------------------------------------------------
  1. Native C180 (recommended)
  2. Native 0.5x0.625 
  3. Pre-processed C180 (not yet available)
  4. Pre-processed 0.5x0.625 (not yet available)
```
There are missing files error for both 1 and 2 and I traced back where the bugs lie. They orginated in default ExtData.rc when creating run directory.

#### Native C180

```
0000: CAP.EXTDATA: ERROR: UpdateBracketTime: Could not find data within maximum offset range from ./HcoDir/OFFLINE_DUST/v2021-08/C180/%y4/%m2/dust_emissions_${RUNDIR_MET_LAT_RES}.%y4%m2%d2.nc
```
- Nonexistent `OFFLINE_DUST/v2021-08/C180` directory for all offline emission (`C180` in place of environment variable `${RUNDIR_MET_NATIVE_RES}`)
- Nonexistent environment variable`${RUNDIR_MET_LAT_RES}`

Referenced in [`geos-chem/run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem`](https://github.com/geoschem/geos-chem/blob/b288af1563a9a49a154a93bdd0f88be1f58f3f2f/run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem#L1603)

Can be fixed by editting [`geos-chem/run/shared/settings/geosit/geosit.native_cs.txt`](https://github.com/geoschem/geos-chem/blob/b288af1563a9a49a154a93bdd0f88be1f58f3f2f/run/shared/settings/geosit/geosit.native_cs.txt#L8) 
- change `${RUNDIR_MET_NATIVE_RES}` to `0.5x0.625`
- add `${RUNDIR_MET_LAT_RES}` to `05`
- add `${RUNDIR_MET_LON_RES}` to `0625`

I know it is not what it meant to be, because native resolution for meteorological field is C180. But it is also used to determine ExtData path for offline emission. You can only correctly locate offline emission file this way.

#### Native 0.5x0.625

```
pe=00003 FAIL at line=02173 ExtDataGridCompMod.F90 <From ./MetDir/%y4/%m2/%d2/GEOS.it.asm.asm_inst_3hr_glo_L576x361_v72.GEOS5294.%y4-%m2-%d2T%h2%n2Z.nc4 could not find file with extrapolation>
```
- Nonexistent `*Z.nc4` file

Referenced and should be editted in [`geos-chem/run/shared/settings/geosit/geosit.native_ll.txt`](https://github.com/geoschem/geos-chem/blob/b288af1563a9a49a154a93bdd0f88be1f58f3f2f/run/shared/settings/geosit/geosit.native_ll.txt#L32)

- all `*Z.nc4` file should be replaced by `*.V01.nc4` file.

#### Reference data directory

Input data directory is [http://geoschemdata.wustl.edu/ExtData](http://geoschemdata.wustl.edu/ExtData/)

- `./HcoDir` is `/ExtData/HEMCO`
- `./MetDir` is `/ExtData/GEOS_C180/GEOS_IT_Native` for Native C180
- `./MetDir` is `/ExtData/GEOS_0.5x0.625/GEOS_IT_Native` for Native 0.5x0.625

**Comment by lizziel** [comment]
Hi @yuanjianz, thanks for trying out GEOS-IT and figuring out solutions to these bugs! We can fix this in the next version.

**Comment by lizziel** [comment]
These are now merged and will be included in 14.3.1.
